index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=``">
+    
+    <script
+			  src="https://code.jquery.com/jquery-3.4.1.min.js"
+			  integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
+			  crossorigin="anonymous"></script>
+    <title>CSS Editor</title>
+</head>
+<body>
+    <label for="csseditor">Type CSS Here:</label>
+    <textarea name="csseditor" class="myTextarea" cols="100" rows="30"></textarea>
+
+    <h1>You can only change the color of these elements lol, This is a h1</h1>
+    <h2>This is a h2</h2>
+    <p>This is a p</p>
+    <script src="script.js"></script>
```

Comment:
Your project has just the basic elements and that's good because it's simple.
I would have loved to see more styling, though. Right now it looks quite sad.